src/services/BoardService.test.js

```diff
@@ -0,0 +1,65 @@
+import { expect, it } from '@jest/globals';
+import { addNewCard } from './BoardService';
+
+it('returns a new board with the new card added', () => {
+  const board = {
```

Comment:
Maybe you can extract that board as a variable and use a `beforeEach` function, reassigning it before each test so it does not have any side effects. 

Moreover, you don't really need the title and the content of the cards since you are only checking the length of the array.

Reply:
not sure what you want to do here, need more explanation